style.css

```diff
@@ -323,4 +324,117 @@
     letter-spacing: 0;
     text-align: center;
   }
+
+  .contact-section {
+    margin: 0 24px;
+    padding-top: 80px;
+  }
+
+  .para-info {
+    width: 327px;
+    height: 216px;
+    font-family: Inter, sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    text-align: center;
+    color: #172b4d;
+    margin-left: 50px;
+  }
+
+  form {
+    margin-top: 150px;
+    margin-left: -50px;
+  }
+
```

Comment:
- [ ] [REQUIRED] Good job so far! Please remove the `margin` in the `contact section` because there is a lot of `space` on the `left` side of the web page and the elements there are sticking to the `right` side. Please make it as shown in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/212970653-20d2c713-e24a-4a3e-8b2c-6ff7542ae5a6.png)